controller/rancher/rancher.go

```diff
@@ -117,8 +134,17 @@ func (lbc *LoadBalancerController) Run(provider provider.LBProvider, metadataURL
 		MetadataClient: metadataClient,
 	}
 
-	lbc.MetaFetcher.OnChange(5, lbc.ScheduleApplyConfig)
+	lbSvc, err := lbc.MetaFetcher.GetSelfService()
+	if err != nil {
+		logrus.Errorf("Error reading self service metadata: %v", err)
+		lbc.Stop()
+	}
+	certDir := lbSvc.Labels["io.rancher.lb_service.cert_dir"]
```

Comment:
as certDir/defaultCertDir are CertFetcher's fields, we should set them in a lb.CertFetcher=<value> manner. Then there will be no need to pass them to the CertFetcher.LookForCertUpdates method.